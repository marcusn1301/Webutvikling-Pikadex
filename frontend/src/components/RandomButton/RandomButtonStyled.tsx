import styled from "styled-components";

export const RandomButtonStyle = styled.div`
  position: absolute;
  top: 12vh;
  left: 85vw;
`;

export const Pokeball = styled.img`
  height: 30px;
  width: 30px;
`;
