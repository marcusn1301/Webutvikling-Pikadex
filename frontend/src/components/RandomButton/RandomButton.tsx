import logo from "../../assets/pokeball.png";
import { Pokeball, RandomButtonStyle } from "./RandomButtonStyled";

const RandomButton = () => {
  return (
    <>
      <RandomButtonStyle>
        <Pokeball src={logo} />
      </RandomButtonStyle>
    </>
  );
};

export default RandomButton;
