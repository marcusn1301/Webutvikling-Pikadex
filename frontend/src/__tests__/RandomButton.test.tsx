import renderer from "react-test-renderer";
import RandomButton from "../components/RandomButton/RandomButton";

test("Snapshot test for Navbar", () => {
  const container = renderer.create(<RandomButton />);
  expect(container).toMatchSnapshot();
});
