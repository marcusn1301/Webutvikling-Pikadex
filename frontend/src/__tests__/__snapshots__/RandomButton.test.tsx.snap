// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Navbar 1`] = `
<div
  className="sc-bcXHqe iAxeXF"
>
  <img
    className="sc-gswNZR jHjZRc"
    src="pokeball.png"
  />
</div>
`;
